.github/ISSUE_TEMPLATE.md

```diff
@@ -1,5 +1,5 @@
-**Note: FOR INTERNAL USE ONLY. For support questions, please use the EDA Chatter group in [The Power of Us Hub](https://powerofus.force.com).
-For feature requests, please use the Ideas tab in [The Power of Us Hub](https://powerofus.force.com).**
+**Note: FOR INTERNAL USE ONLY. For support questions, please use the EDA Chatter group in The Power of Us Hub.
+For feature requests, please use the Ideas tab in The Power of Us Hub.**
```

Comment:
@mrrigney I would remove this line.

Reply:
Line 2, that is.

Reply:
@aarsvoboda I modified this line, can you please verify?  Made you an approver